src/ssSeasonsUtil.lua

```diff
@@ -126,6 +127,29 @@ function ssSeasonsUtil:year(dayNumber)
     return math.floor((dayNumber - 1) / (self.daysInSeason * self.seasonsInYear))
 end
 
+-- Get the InGame date in for seasons and InGame day
```

Comment:
I know that there are no months at the moment. But to show the user / player where he is standing in the year, I thought it would be a small addition, the function is used in the ssWeatherForcast.lua. But this is not really an important point now, as I said, just an small addition.